### ISSUE TITLE
Ln 123: TypeError: string indices must be integers

### ISSUE THREAD

**Issue by anewbegin95** [anewbegin95/personalfinance#3]
Running the following step:

`for ticker in tickers:
    adjCloseDict[ticker] = load_csv_data(ticker)[int(0)]['Adj Close']`

Results in the error in the title above. I believe there's somewhere that I'm setting an index in the load_csv_data function, but I'm not sure where.

**Comment by Jim89** [comment]
It's hard to diagnose this without seeing the full context of the program (i.e. without an reproducible example). In the mean time, I would try to satisfy yourself that:

* You can successfully load from csv for all values of `ticker`, to make sure that your `load_` function does what you expect. e.g. try:
```
for ticker in tickers:
    print(ticker)
    load_csv_date(ticker)
```
* You can successfully subset the loaded data using `[int(0)]`
* You can successfully subset _that_ subset using [`Adj Close']
* `adjCloseDict[ticker]` (i.e subsetting/assigning to your dictionary) works for all of the `ticker` values in `tickers` (i.e. that the index error isn't in dictionary subsetting).

Basically, unwrap the code a bit and figure out which step the bug is triggering at. There are probably some debug tools in your editor/in Python that we can have a look at, too, I just don't know what they are off the top of my head.
